certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          }
+        },
+        "realtor": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "attorney": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "tax1099": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Irs1099"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TransactionParties": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sellers": {
+          "description": "Gets or Sets Sellers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Seller"
+          }
+        },
+        "buyers": {
+          "description": "Gets or Sets Buyers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Buyer"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TransactionType": {
+      "format": "int32",
+      "description": "Gets or Sets TransactionType",
+      "enum": [
+        0,
+        1,
+        2
+      ],
+      "type": "integer"
+    },
+    "PubCS.TitleOrder.Api.Models.LoanPayment": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "principal": {
+          "format": "double",
+          "description": "Gets or Sets Principal",
+          "type": "number"
+        },
+        "interest": {
+          "format": "double",
+          "description": "Gets or Sets Interest",
+          "type": "number"
+        },
+        "mortgageInsurance": {
+          "format": "double",
+          "description": "Gets or Sets MortgageInsurance",
+          "type": "number"
+        },
+        "taxes": {
+          "format": "double",
+          "description": "Gets or Sets Taxes",
+          "type": "number"
+        },
+        "hoa": {
+          "format": "double",
+          "description": "Gets or Sets Hoa",
+          "type": "number"
+        },
+        "totalMonthly": {
+          "format": "double",
+          "description": "Gets or Sets TotalMonthly",
+          "type": "number"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.LoanDetail": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "rate": {
+          "format": "double",
+          "description": "Gets or Sets Rate",
+          "type": "number"
+        },
+        "apr": {
+          "format": "double",
+          "description": "Gets or Sets Apr",
+          "type": "number"
+        },
+        "term": {
+          "format": "int32",
+          "description": "Gets or Sets Term",
+          "type": "integer"
+        },
+        "payment": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.LoanPayment"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Loan1": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.